http-agent/src/main/java/org/motechproject/http/agent/constants/SendRequestConstants.java

```diff
@@ -0,0 +1,18 @@
+package org.motechproject.http.agent.constants;
+
+/**
+ * The <code>SendRequestConstants</code> contains constant values used in the http-agent module.
+ */
+
+public final class SendRequestConstants {
+    public static final String SEND_REQUEST_SUBJECT = "SendRequest";
+    public static final String URL = "URL";
+    public static final String USERNAME = "Username";
+    public static final String PASSWORD = "Password";
+    public static final String FOLLOW_REDIRECTS = "FollowRedirects";
+    public static final String DATA = "BodyParameters";
```

Comment:
Either rename the static name to BODY_PARAMETERS or change the string to "Data".